go/engine/soft_snooze_test.go

```diff
@@ -140,6 +142,9 @@ func TestSoftSnooze(t *testing.T) {
 	if pe := libkb.ImportProofError(result.ProofResult); pe == nil {
 		t.Fatal("expected a Rooter error result")
 	}
+	if !idUI.BrokenTracking {
```

Comment:
We expect the bool to be set. I tested that this check breaks before the bugfix below.